src/main/webapp/dev/js/main/instructorCourseEnrollPage.js

```diff
@@ -42,8 +42,118 @@ if (window.addEventListener) {
     window.attachEvent('load', loadUpFunction);
 }
 
+const container = document.getElementById('spreadsheet');
+
+/**
+ * Holds Handsontable settings, reference and other information for Spreadsheet.
+ */
+/* global Handsontable:false */
+const handsontable = new Handsontable(container, {
+    manualColumnResize: true,
+    manualRowResize: true,
+    manualColumnMove: true,
+    rowHeaders: true,
+    colHeaders: ['Section', 'Team', 'Name', 'Email', 'Comments'],
+    columnSorting: true,
+    className: 'htCenter',
+    sortIndicator: true,
+    maxCols: 5,
+    maxRows: 100,
+    stretchH: 'all',
+    minSpareRows: 1,
+    contextMenu: [
+        'row_above',
+        'row_below',
+        'remove_row',
+        'undo',
+        'redo',
+        'make_read_only',
+        'alignment',
+    ],
+});
+
+/**
+ * Updates column header order and generates a header string.
+ *
+ * Example: Change this array ['Section', 'Team', 'Name', 'Email', 'Comments']
+ * into a string = "Section | Team | Name | Email | Comments"
+ */
+function updateHeaderOrder() {
```

Comment:
I think this function should be named `getUpdatedHeaderString`. Initially I was confused because looking at the code, there's no updating action going on.